src/customize-pop/customize-pop.md

```diff
@@ -0,0 +1,33 @@
+<!--sources:
+Sunrise Pop Training (google doc)
+https://support.system76.com/articles/customize-gnome
+-->
+
+# Customize Pop!\_OS
+
+Users can customize the look and feel of Pop!\_OS in the `Settings` menu. The GNOME Tweaks and Extensions applications provide customization beyond the `Settings` menu.
+
+- [Customize the Desktop](/customize-pop/customize-desktop.md) - Choose background wallpaper, set a dark or light theme, modify Dock appearance, and modify workspace behavior.
+- [Configure Application Settings](/customize-pop/application-settings.md) - Specify which applications are allowed to display on-screen messages, and designate application to handle specific file types.
+- [Change Keyboard Settings](/customize-pop/keyboard-settings.md) - Change your keyboard language, layout, and customize keyboard shortcuts.
+- [Connect an Online Account](/customize-pop/online-accounts.md) - Synchronize your files by connecting an online account with an existing cloud storage provider.
+- [Assign Color Profiles](/customize-pop/color-profiles.md) - Activate or upload color profiles for individual displays and users.
+- [Customize Region and Language Settings](/customize-pop/region-language.md) - Manage installed languages and formatting for numbers, dates, time, and currencies.
```

Comment:
I understand that you're referring to the concept of "time" as opposed to various times, but you may want to consider making that plural to match the rest of the list.